browser/ui/webui/brave_new_tab_page_refresh/new_tab_page_handler.cc

```diff
@@ -366,12 +366,34 @@ void NewTabPageHandler::SetShowClock(bool show_clock,
 }
 
 void NewTabPageHandler::GetClockFormat(GetClockFormatCallback callback) {
-  std::move(callback).Run(pref_service_->GetString(kNewTabPageClockFormat));
+  auto to_clock_format = [](const std::string& format_string) {
```

Comment:
@fallaciousreasoning Let me know if there's a different way you'd prefer this enum-to-pref conversion done.

Reply:
yeah, don't think we can do anything else here :smile: The main thing I'd like to change (in a follow up) is to also add `kNone` to the enum